reservation/reservation.go

```diff
@@ -74,9 +76,11 @@ func (manager *Manager) Lock(resource string) (*Reservation, error) {
 		"EX", manager.TTL.Seconds(),
 		"NX")
 	if err != nil {
+		manager.lg.ErrorD("redis-error", logger.M{"key": key, "resource": resource, "err": err.Error()})
 		return nil, fmt.Errorf("Error with SET command: %s", err.Error())
 	}
 	if success == nil {
+		manager.lg.InfoD("reservation-exists", logger.M{"key": key, "resource": resource})
```

Comment:
`InfoD` mostly to match the node implementation: https://github.com/Clever/node-redis-reservation/blob/95eb69186b7c31ed60689123b45e7d1dca717e7c/lib/index.coffee#L41

I'm not in love with this API but it isn't worth switching it up either IMO.